### ISSUE TITLE
Kernel tests fail after release of Drupal 10.2

### ISSUE THREAD

**Issue by darrenoh** [retrofit-drupal/retrofit#140]
For some reason kernel tests try to parse sqlite.info.yml before changing the working directory to the app root, which of course means the file cannot be found. In Drupal 10.1, this just returned an empty array. In 10.2, it throws an exception, causing kernel tests to fail.

**Comment by darrenoh** [comment]
A temporary fix could be to pin Drupal to 10.1 for testing until we determine how to fix.

**Comment by mglaman** [comment]
Well this is annoying.

```php
    /** @var \Drupal\Core\Extension\DatabaseDriver $driver */
    $driver = self::getDriverList()
      ->includeTestDrivers($include_test_drivers)
      ->get($driverNamespace);
```

This is another version of an extension list. In `\Drupal\Core\Database\Database::getDriverList` it hardcodes the root:

```
      return new DatabaseDriverList(DRUPAL_ROOT, 'database_driver', new NullBackend('database_driver'));
```
